cspell.json

```diff
@@ -51,34 +43,53 @@
         {
             "languageId": "markdown",
             "words": [
-                "SUBDIR",
                 "rustup",
+                "SUBDIR",
                 "Yocto"
             ],
             "ignoreRegExpList": []
         }
     ],
     "words": [
+        "analysing",
```

Comment:
```suggestion
        "analysing", // deliberate choice of British spelling here
```

Reply:
Maybe worth a comment or changing to US spelling?

Reply:
Oh right! That's british spelling. I totally did not have that on the radar and was stumped why this was flagged as wrong!

Let's stick with US spelling, I'll remove this again.